test/src/rabbit_web_mqtt_SUITE.erl

```diff
@@ -14,13 +14,58 @@
 %%   Copyright (c) 2007-2016 Pivotal Software, Inc.  All rights reserved.
 %%
 
--module(rabbit_web_mqtt_test).
+-module(rabbit_web_mqtt_SUITE).
 
 -include_lib("eunit/include/eunit.hrl").
 -include("emqttc_packet.hrl").
 
-connection_test() ->
-    WS = rfc6455_client:new("ws://127.0.0.1:15675/ws", self()),
+-compile(export_all).
+
+all() ->
+    [
+      {group, non_parallel_tests}
+    ].
+
+groups() ->
+    [
+      {non_parallel_tests, [],
+       [connection_test
+        ,pubsub_test
+        ,disconnect_test
```

Comment:
Please remove the `_test` suffix so they are not to be confused with EUnit tests.